yrs-python/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+use pyo3::prelude::*;
+use pyo3::wrap_pyfunction;
+use pyo3::types::{PyAny };
+use pythonize::{depythonize, pythonize};
+use yrs_api_wrapper;
+
+
+#[pyfunction]
+pub fn merge_updates(updates:Vec<Vec<u8>> ) -> PyResult<Py<PyAny>> {
+ 
+    // Converts a Vec<Vec<u8>>  into a   [&[u8]]
+    let updates_u8 : Vec<&[u8]> = updates.iter().map(|x| &x[..]).collect();
```

Comment:
That Was my fault. The definition of `merge_updates` should have been `merge_updates(&[Vec<u8>])`. @Horusiath should we switch the interface to `std::io::Read` instead or keep using the `Vec<u8>` definition for updates for now.